dxfeed/core/DXFeedPy.pyx

```diff
@@ -53,24 +53,30 @@ cdef class ConnectionClass:
     Data structure that contains connection
     """
     cdef clib.dxf_connection_t connection
-    # sub_ptr_list contains pointers to all subscriptions related to current connection
-    cdef cppdq[clib.dxf_subscription_t *] sub_ptr_list
-    # each subscription has its own index in a list
-    cdef int subs_order
+    cdef object __sub_refs
 
     def __init__(self):
-        self.subs_order = 0
+        self.__sub_refs = WeakSet()
 
     def __dealloc__(self):
         dxf_close_connection(self)
 
+    def get_sub_refs(self):
+        """
+        Method to get list of references to all subscriptions related to current connection
+
+        Returns
+        -------
+        :list
+            List of weakref objects. Empty list if no refs
+        """
+        refs = list(self.__sub_refs) if self.__sub_refs else list()
```

Comment:
For what purposes do we need this ternary operator here? If we'll just pass empty `WeakSet` to `list()` it'll give us just `[]` as output. 🤔